easybuild/easyconfigs/m/Meson/Meson-0.48.1-fosscuda-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'Meson'
+version = '0.48.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://mesonbuild.com'
+description = "Meson is a cross-platform build system designed to be both as fast and as user friendly as possible."
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ef893e2d64083463fe41f05d683edbe8c44bc187cd9cc66cbc2b1ce399567447']
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Ninja', '1.8.2'),
+]
+
```

Comment:
@akesandgren Missing `download_dep_fail = True` and `use_pip = True`

Reply:
I blame the foss-2018b version in develop...